README.md

```diff
@@ -477,6 +479,13 @@ obj = $form.find(':input').filter(function () {
       }).serializeJSON();
 ```
 
+Or you can use any of the following options:
+
+ * `skipFalsyValuesForTypes` option to skip empty values for listed types
+ * `skipFalsyValuesForFields` option to skip empty values for listed field names
+ * `data-skip-falsy="true"` input field attribute to skip empty values on the current field
+
+_Note: To make it work correctly you should use it on fields with :type or set the `parseAll` option to `true`._
```

Comment:
No need to explain the options here again, you can just make a quick to the explanation before:

```
For falsy values (`false, "", 0, null, undefined, NaN`), you can use serializeJSON specific options to skip fields by name (`skipFalsyValuesForFields: [name]`) or by type (`skipFalsyValuesForTypes: ["string", "null"]`).

You can also use a data attribute `data-skip-falsy="true"` on the inputs that should be ignored (`data-skip-falsy` is aware of field :types, so it knows how to skip a non-empty input like this `<input name="foo:number" value="0" data-skip-falsy="true">`).
```